mm2src/common/for_tests.rs

```diff
@@ -216,7 +216,12 @@ impl MarketMakerIt {
     /// * `local` - Function to start the MarketMaker in a local thread, instead of spawning a process.
     /// It's required to manually add 127.0.0.* IPs aliases on Mac to make it properly work.
     /// cf. https://superuser.com/a/458877, https://superuser.com/a/635327
-    pub fn start(mut conf: Json, userpass: String, local: Option<LocalStart>) -> Result<MarketMakerIt, String> {
+    pub fn start(
+        mut conf: Json,
+        userpass: String,
+        local: Option<LocalStart>,
+        envs: &[(&str, &str)],
+    ) -> Result<MarketMakerIt, String> {
```

Comment:
What about adding a `fn with_envs` or `fn start_with_envs`. It would allow us not to pass an empty slice when there is no need to pass anything.
Then the `start` function would look like this:
```rust
pub fn start(conf: Json, userpass: String, local: Option<LocalStart>) -> Result<MarketMakerIt, String> {
    MarketMakerIt::start_with_envs(conf, userpass, local, &[])
}
```

Reply:
Yeah, I like your idea, let me do it :slightly_smiling_face:

Reply:
Done, please recheck :slightly_smiling_face: